backend/medtagger/repositories/tasks.py

```diff
@@ -0,0 +1,65 @@
+"""Module responsible for definition of TaskRepository."""
+from typing import List
+
+from medtagger.database import db_session
+from medtagger.database.models import Task, LabelTag, scan_categories_tasks
+
+
+def get_all_tasks() -> List[Task]:
+    """Fetch all tasks from database ordered by id."""
+    with db_session() as session:
+        tasks = session.query(Task).order_by(Task.id).all()
+    return tasks
+
+
+def get_task_by_key(key: str) -> Task:
+    """Fetch Task from database.
+
+    :param key: key for a Task
+    :return: Task object
+    """
+    with db_session() as session:
+        task = session.query(Task).filter(Task.key == key).one()
+    return task
+
+
+def add_task(key: str, name: str, image_path: str, categories_ids: List[int]) -> Task:
+    """Add new Task to the database.
+
+    :param key: key that will identify such Task
+    :param name: name that will be used in the Use Interface for such Task
+    :param image_path: path to the image that represents such Task (used in User Interface)
+    :param categories_ids: IDs of ScanCategories that Task takes Scans from
+    :return: Task object
+    """
+    with db_session() as session:
+        task = Task(key, name, image_path)
+        session.add(task)
+        for category_id in categories_ids:
+            # pylint: disable=no-value-for-parameter
+            scan_categories_tasks.insert().values(scan_cateogory_id=category_id, task_id=task.id)
```

Comment:
This may not work due the fact that `task.id` will be None until transactions' commit will be called (hidden in `db_session()`).

What's more, I think that such code could be written this way:
```python
with db_session() as session:
    task = Task(key, name, image_path)
    scan_categories = ScanCategory.query.filter(ScanCategory.id.in_(categories_ids)).all()
    task.scan_categories = scan_categories
    session.add(task)
```

...but I didn't test it at all 😉

Or maybe this way? I don't know...
```python
with db_session() as session:
    task = Task(key, name, image_path)
    scan_categories = ScanCategory.query.filter(ScanCategory.id.in_(categories_ids)).all()
    task.scan_categories.extend(scan_categories)
    session.add(task)
```